MANIFEST.in

```diff
@@ -1,5 +1 @@
-include caio/*.pyi
-include caio/py.typed
 recursive-include caio/src/threadpool *.*
-include README.rst
-include LICENSE
```

Comment:
@cdce8p Please do not remove license files from packaging. This is bad practice in general and breaks any downstream packaging systems, like conda-forge.

This needs to get reverted.

Reply:
Ah okay, just renamed for some reason: https://github.com/mosquito/caio/pull/41

So no need to revert, just confused by choices PR #41.